summarize/src/main.rs

```diff
@@ -192,6 +203,7 @@ fn summarize(opt: SummarizeOpt) -> Result<(), Box<dyn Error + Send + Sync>> {
         .sort_by(|l, r| r.self_time.cmp(&l.self_time));
 
     let mut table = Table::new();
+    table.set_format(*prettytable::format::consts::FORMAT_NO_LINESEP_WITH_TITLE);
```

Comment:
Missing for the diff command.

Reply:
derp, I should have actually tested the `diff` change.